cherrypy/test/test_httputil.py

```diff
@@ -79,3 +80,82 @@ def test_invalid_status(status_code, error_msg):
     """Check that invalid status cause certain errors."""
     with pytest.raises(ValueError, match=error_msg):
         httputil.valid_status(status_code)
+
+
+@pytest.mark.parametrize(
+    'header_content,value,params',
```

Comment:
I didn't add the corresponding linter here but a better style is having an iterable of strings ­— it reads better because it's sparse.
```suggestion
    ('header_content', 'value', 'params'),
```